.github/workflows/scripts/countline.py

```
@@ -0,0 +1,295 @@
+    Run these commands from the CLI to ensure the code is compliant for all
+    your pull requests.
+
+"""
+# Standard imports
+import os
+import sys
+import argparse
+from collections import namedtuple
+
+
+def _valid_filename(filepath):
```

Ruff (0.12.2)

Missing return type annotation for private function `_valid_filename`

(ANN202)